File: Examples/SimpleExample/SimpleExample/ViewController.m
//
//  ViewController.m
//  SimpleExample
//
//  Created by Elijah Windsor on 12/12/16.
//  Copyright © 2016 Kocomojo. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];

    // button is assigned in LaunchScreen.storyboard.
    [self.button setDisabledImageName:@"kocomojo_disabled"];
    [self.button setEnabledImageName:@"kocomojo"];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}


@end

File: Examples/SimpleExampleSwift/SimpleExampleSwift/ViewController.swift
//
//  ViewController.swift
//  SimpleExampleSwift
//
//  Created by Elijah Windsor on 12/15/16.
//  Copyright © 2016 Kocomojo. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    @IBOutlet weak var button: KocomojoButton?;

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        self.button?.enabledImageName = "kocomojo";
        self.button?.disabledImageName = "kocomojo_disabled";
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

